### PR TITLE
feat: Screenshot KSS styleguide

### PR DESCRIPTION
**Pull request by ptbrowne**
Add screenshots for the KSS styleguide.
Utilities are not screenshotted as their screenshots are huge.

### CODE REVIEW

File: scripts/screenshots.js [hunk 1]
```diff
@@ -352,6 +352,60 @@ const screenshotStackExamples = async (page, args) => {
...
+  await page.setViewport({ width: WIDTH, height: 800})
+  const links = await page.evaluate(() => {
+    const navLinks = Array.from(document.querySelectorAll('.kss-nav > .kss-nav__item > a[href]'))
+    return navLinks.map(node => node.getAttribute('href'))
+  })
+
+  let i =0
+  for (const link of links) {
+    i = i + 1
+    console.log(link)
```
Comment (Crash--) [inline scripts/screenshots.js:375]:
```suggestion
```

File: scripts/screenshots.js [hunk 2]
```diff
@@ -398,8 +457,9 @@ const main = async () => {
   })
   const { browser, page } = await prepareBrowser({ viewport: parsedViewport })
 
-  await screenshotStyleguide(page, args, config)
-  await screenshotStackExamples(page, args)
+  // await screenshotStyleguide(page, args, config)
```
Comment (Crash--) [inline scripts/screenshots.js:460]:
```suggestion
  await screenshotStyleguide(page, args, config)
```

File: scripts/screenshots.js [hunk 3]
```diff
@@ -398,8 +457,9 @@ const main = async () => {
...
+  await screenshotKSSStyleguide(page, args)
+  // await screenshotStackExamples(page, args)
```
Comment (Crash--) [inline scripts/screenshots.js:462]:
```suggestion
  await screenshotStackExamples(page, args)
```

**Review by Crash-- — COMMENTED** [review]
<3

File: scripts/screenshots.js (hunk 1 above)
Comment (ptbrowne) [reply scripts/screenshots.js:375]: Solved

File: scripts/screenshots.js (hunk 3 above)
Comment (ptbrowne) [reply scripts/screenshots.js]: Solved in https://github.com/cozy/cozy-ui/pull/1664/commits/a157bbf7eb5e472fc830f03c241d17d4fd85dd16

File: scripts/screenshots.js (hunk 2 above)
Comment (ptbrowne) [reply scripts/screenshots.js]: Solved in https://github.com/cozy/cozy-ui/pull/1664/commits/a157bbf7eb5e472fc830f03c241d17d4fd85dd16

**Review by JF-Cozy — APPROVED** [review]

**Pull request merged**